package.json

```diff
@@ -54,6 +54,9 @@
   "peerDependencies": {
     "vue": "^3.0.2"
   },
+  "dependencies": {
+    "@vue/devtools-api": "^6.0.0-beta.7"
```

Comment:
If it is added as a dependency, shouldn't we mark it as external too? I will need to follow the same on Vue Router. Currently it is not marked as external and therefor is only a dev dependency

Reply:
I thought this was here to support Type imports in TS...? In that case I think it should be the same in Router too. Am I correct @Akryum ?